src/main/java/org/apache/commons/lang3/CharUtils.java

```diff
@@ -329,11 +332,12 @@ public static String toString(final char ch) {
      * @param ch  the character to convert
      * @return a String containing the one specified character
      */
-    public static String toString(final Character ch) {
+    @SuppressWarnings("index:argument.type.incompatible") // #1: @NonNegative argument as defined in parameter to the function
+    public static String toString(final @NonNegative Character ch) {
         if (ch == null) {
             return null;
         }
-        return toString(ch.charValue());
+        return toString(ch.charValue()); // #1
```

Comment:
Is this an instance of [typetools/checker-framework#2540](https://github.com/typetools/checker-framework/issues/2540) or something very similar? Another GSoC student is working on fixing that right now, so commenting on that issue with a test case with this method in it would be useful. Can you do that?